grammar/scenic.gram

```diff
@@ -0,0 +1,2070 @@
+# PEG grammar for Scenic
```

Comment:
Let's acknowledge here that this grammar is adapted from the Python grammar and include a link to the latter.

Reply:
da576f1b009297e2e175852591cd9a820d9abeee
Added a sentence right below. Let me know if we should include more information (I think we can also discuss this later, but just adding this not to forget).